src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/UIContainerCell.cs

```diff
@@ -31,6 +31,13 @@ internal UIContainerCell(string cellId, View view, Shell shell, object context)
 			var platformView = view.ToPlatform();
 			ContentView.AddSubview(platformView);
 			platformView.AccessibilityTraits |= UIAccessibilityTrait.Button;
+			platformView.TranslatesAutoresizingMaskIntoConstraints = false;
+
+			var margin = view.Margin;
+			platformView.LeadingAnchor.ConstraintEqualTo(ContentView.LeadingAnchor, (nfloat)margin.Left).Active = true;
```

Comment:
Can you create a `NSLayoutConstraint[]` with all the constraints and then use the method `NSLayoutConstraint.ActivateConstraints(constraints);` to active all of them at once? Is more efficient than  individually.

Reply:
@jsuarezruiz, I have updated the code based on your suggestion. Could you please share your concerns on this.